packages/components/src/styles/table-cell/table-cell.stories.ts

```diff
@@ -0,0 +1,275 @@
+        'sd-table-cell--transparent',
+        'sd-table-cell--primary-100',
+        'sd-table-cell--neutral-100'
+      ]
+    }
+  },
+  render: (args: Record<string, any>) => {
+    // Initalize table data
+    const tableRowCount = 7;
+    const tableColumnCount = 5;
+    const sortData: ('ascending' | 'descending' | 'none')[] = Array.from({ length: tableColumnCount }, () => 'none');
+    const headerData = Array.from({ length: tableColumnCount }, () => 'Header');
+    const tableData = Array.from({ length: tableRowCount }, () =>
+      Array.from({ length: tableColumnCount }, () => {
+        return args['default-slot'] ?? 'Lorem ipsum dolor sit amet.';
+      })
+    );
+
+    // Function to exchange the sort icons and set the aria-sort attribute in the sortable table after a click on the table header
+    const sortTable = (column: number) => {
+      const icon = document.getElementById(`sortIcon-${column}`);
+      const headerCell = document.getElementById(`sortableHeader-${column}`);
+      if (icon && headerCell) {
+        switch (sortData[column]) {
+          case 'none':
+            sortData[column] = 'descending';
+            icon.setAttribute('name', 'system/sort-down-filled');
+            headerCell.setAttribute('aria-sort', 'descending');
+            return;
+          case 'ascending':
+            sortData[column] = 'none';
+            icon.setAttribute('name', 'system/sort-down');
+            headerCell.removeAttribute('aria-sort');
+            return;
+          case 'descending':
+            sortData[column] = 'ascending';
+            icon.setAttribute('name', 'system/sort-up-filled');
+            headerCell.setAttribute('aria-sort', 'ascending');
+        }
+      }
+    };
+
+    return html`
+      <style>
+        .story-wrapper {
+          display: inline-block;
+          max-width: 1200px;
+        }
+
+        .headline {
+          padding: 16px;
+          background: #e0e0e0;
+          text-align: left;
+          font-size: 14px;
+          font-weight: bold;
+          width: 100%;
+          box-sizing: border-box;
+        }
+
+        .disclaimer {
+          background-color: #f6f6f6;
+          padding: 16px;
+          text-align: left;
+          font-size: 14px;
+          color: #333;
+          width: 100%;
+          margin-bottom: 10px;
+          box-sizing: border-box;
+        }
+
+        .sd-table {
+          margin-top: 20px;
+          width: 100%;
+        }
+
+        .sd-table-cell.sortable {
+          padding: 0;
+        }
+
+        .sd-table-cell.sortable button {
+          display: inline-flex;
+          box-sizing: border-box;
+          height: 100%;
+          width: 100%;
+          margin: 0;
+          padding: 16px;
+          align-items: center;
+          justify-content: flex-start;
+          gap: 16px;
+          text-align: left;
+        }
+
+        .sd-table-cell.sortable button:hover {
+          background-color: white;
```

Comment:
It would be helpful to make this visible in the story's code.

Reply:
Maybe we can also replace the CSS part with our tokens...?

Reply:
I wanted to do that, but I could not find out how to use our tokens in storybook. Do you know how to do that?